packages/mobile/src/components/share-drawer/useShareToStory.tsx

```diff
@@ -356,10 +358,9 @@ export const useShareToStory = ({
         // For simplicity, assume that calculating dominant colors and generating the sticker takes 20% of the total loading time:
         dispatch(setProgress(20))
 
-        const encodedTrackId = encodeHashId(content.track.track_id)
-        const streamMp3Url = apiClient.makeUrl(
-          `/tracks/${encodedTrackId}/stream`
-        )
+        const streamMp3Url = await sdk.tracks.getTrackStreamUrl({
+          trackId: Id.parse(content.track.track_id)
```

Comment:
Does this need current user? I know the original didn't have it, but maybe it injected it.